WebAutopark/Startup.cs

```diff
@@ -23,6 +26,10 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            string connectionString = Configuration.GetConnectionString("DataBase");
+            services.AddTransient<IRepository<VehicleType>, VehicleTypesRepository>(provider => new VehicleTypesRepository(connectionString));
+            services.AddTransient<IRepository<Vehicle>, VehicleRepository>(provider => new VehicleRepository(connectionString));
+            services.AddTransient<IRepository<SparePart>, SparePartsRepository>(provider => new SparePartsRepository(connectionString));
```

Comment:
Use Scoped lifetime for repositories.

Reply:
As per agreement with Konstantin, I will perform it later after learning the appropriate topics